aqovia-nuget-update.ps1

```diff
@@ -0,0 +1,152 @@
+﻿[CmdletBinding()]
+Param(
+   [Parameter(Mandatory=$True,Position=1)]
+   [string]$packageName,
+	
+   [Parameter(Mandatory=$True)]
+   [string]$targetVersion,
+	
+   [Parameter(Mandatory=$True)]
+   [string]$branchName
+)
+
+$hasUpdate = $false
+$workingDir = $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath('.\')
+
+#msbuild path
+$MsBuildExe = Resolve-Path "${env:ProgramFiles(x86)}\Microsoft Visual Studio*\MSBuild\*\bin\msbuild.exe" -ErrorAction SilentlyContinue
+
+#install nuget.exe
+$sourceNugetExe = "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
+$targetNugetExe = "$workingDir\nuget.exe"
+if(-not (Test-Path -path $targetNugetExe)){
+    Invoke-WebRequest $sourceNugetExe -OutFile $targetNugetExe
+    Set-Alias nuget $targetNugetExe -Scope Global -Verbose
+}
+
+#install xunit runner console.exe
+$xunitConsoleExe = $workingDir+"\xunit.runner.console.*\tools\net452\xunit.console.x86.exe"
```

Comment:
If time, maybe search for the exe / frameworkversion rather than assuming net452